src/www/status_dhcp_leases.php

```diff
@@ -48,9 +48,11 @@ function adjust_gmt($dt)
     }
 
     foreach ($dhcpd as $dhcpditem) {
-        $dhcpleaseinlocaltime = $dhcpditem['dhcpleaseinlocaltime'];
-        if ($dhcpleaseinlocaltime == "yes") {
-            break;
+        if (isset($dhcpditem['dhcpleaseinlocaltime'])) {
```

Comment:
probably better to replace with isset($dhcpditem['dhcpleaseinlocaltime']) && $dhcpditem['dhcpleaseinlocaltime'] == "yes"  and ditch $dhcpleaseinlocaltime